book/zh-cn/09-others.md

```diff
@@ -179,6 +179,56 @@ int main() {
 其中 `std::max_align_t` 要求每个标量类型的对齐方式严格一样，因此它几乎是最大标量没有差异，
 进而大部分平台上得到的结果为 `long double`，因此我们这里得到的 `AlignasStorage` 的对齐要求是 8 或 16。
 
+## 9.5 字符串与数值转换
+
+### 数值转字符串
+
+- std::to_string(val)
```

Comment:
please add more explanation about the function, also address the importance of having this in the tutorial.